### ISSUE TITLE
Make it easier for people using assistive tech to sign in

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#600]
## Problem statement
[Sign in button on the top right of the Notify website was where participants expected, but was a bit difficult to locate/see because the font was a bit small for people using magnification.
](https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwBYyaQxOttZ0IAw/recBn563dIs4qcI3g?blocks=hide)

["Click sign in" Doesn't work as a verbal prompt on the homepage (using Dragon Naturally Speaking)
](https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwEUFgBd0KLqH7oo/recV5AigUJXIvjHyt?blocks=hide)

## Hypothesis
Making the sign in button on the top right of the Notify bigger and respond to verbal prompt will make it easier for people who use assistive tech to have more confidence in what they're clicking and have an easier time signing in.
9
## QA Steps
- [ ]  "Sign in" link text is enlarged on both Desktop and Mobile viewports
- [ ]  "Sign in" link has an aria label 
- [ ]  Language toggle link text is enlarged on both Desktop and Mobile viewports
- [ ]  Once logged in, the "Your Account" menu link and sub items are enlarged
- [ ]  Once logged in, using a mobile viewport size, the menu items for "Your account" should be enlarged and match the font size of the items in the "Menu" button
- [ ]

**Comment by whabanks** [comment]
Documentation around Dragon naturally speaking targeted at developers is sparse. Based off [this document](https://www.yumpu.com/en/document/read/3266376/guidelines-for-speech-accessible-html-for-dragon-nuance), they recommend adding an `alt` attribute to `anchor` tags. However this produces non-conforming HTML. Instead an `aria-label` [was added to the sign in link](https://github.com/cds-snc/notification-admin/pull/1574) top right to ensure proper association between the link and the displayed text.

Another point to consider from that document is that the software only processes the first 200 elements of a page. This limit can be increased but without more information about the setup used by the individual who pointed this out, it's difficult to confidently ascertain if this is an issue on our end or with the voice recognition software.

**Comment by andrewleith** [comment]
- Finished up code-wise; ready for QA!

**Comment by adriannelee** [comment]
Done QA, lgtm! @amazingphilippe and I are going to look into getting Dragon so we can do some manual testing more generally too. We can revisit this then to test.
